OmsAgent/omsagent.py

```diff
@@ -17,6 +17,9 @@
 # limitations under the License.
 
 from __future__ import print_function
+from future import standard_library
```

Comment:
Is future module included in the base python?

Reply:
it is part of the standard library in 2.6+ and 3, but I don't think all distros have it ... I'll have to install it in the shim.